app/actors/hyrax/actors/ensure_well_formed_admin_set_actor.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+module Hyrax
+  module Actors
+    # This "Actor" ensures that we have a well formed AdminSet.
+    #
+    # @note For historical reasons, we lazily apply the default admin
+    #   set to curation concerns that don't already have an admin set.
+    #
+    # @see AdminSet
+    # @see Hyrax::PermissionTemplate
+    # @see Hyrax::Actors::DefaultAdminSetActor
+    module EnsureWellFormedAdminSetActor
```

Comment:
should this be moved outside `hyrax/actors` and renamed something else? it doesn't conform to the actor interface.

Reply:
I went ahead and moved the one I created.